include/mxnet/executor.h

```diff
@@ -152,19 +152,19 @@ class Executor {
   static Executor* SimpleBind(nnvm::Symbol symbol,
                               const Context& default_ctx,
                               const std::map<std::string, Context>& group2ctx,
-                              const std::vector<Context>& in_arg_ctxes,
-                              const std::vector<Context>& arg_grad_ctxes,
-                              const std::vector<Context>& aux_state_ctxes,
-                              const std::unordered_map<std::string, TShape>& arg_shape_map,
-                              const std::unordered_map<std::string, int>& arg_dtype_map,
-                              const std::unordered_map<std::string, int>& arg_stype_map,
-                              const std::vector<OpReqType>& grad_req_types,
-                              const std::unordered_set<std::string>& param_names,
+                              std::vector<Context>* in_arg_ctxes,
```

Comment:
@reminisce  Because if things are to be mutated, they need to be pointers, not non-const references (per the linter rules). Given your earlier comments about SimpleBindEx rather than modifying SimpleBind, this will be addressed there rather than modifying it here.